addons/extension/XEH_preInit.sqf

```diff
@@ -0,0 +1,27 @@
+#include "script_component.hpp"
+ADDON = false;
+
+if (!isServer) exitWith {};
+
+PREP_RECOMPILE_START;
+#include "XEH_PREP.hpp"
+PREP_RECOMPILE_END;
+
+GVAR(enabled) = EXT callExtension "setup" == "true";
+GVAR(handlers) = createHashMap;
+
+addMissionEventHandler ["ExtensionCallback", {
+    params ["_name", "_function", "_data"];
+
+    // TRACE_3("Extension callback",_name,_function,_data);
```

Comment:
Why is this commented? If debug mode is off this would not work anyway.

Reply:
It's too noisy even for dev mode, left it so it's easily seen that it might be worth debugging this part in case of issues.